scripts/pre_commit_linter.py

```diff
@@ -1044,7 +1048,6 @@ def _check_docstrings(all_files):
                     # Case 2: line contains some words before """. """ should
                     # shift to next line.
                     elif not any(word in line for word in EXCLUDED_PHRASES):
-                        failed = True
```

Comment:
Just to check, was removing this intentional?

Reply:
Hmm. I think I missed this. This shouldn't be removed.

Reply:
Done.